apps/desktop/src/autofill/services/desktop-fido2-user-interface.service.ts

```diff
@@ -195,42 +210,43 @@ export class DesktopFido2UserInterfaceSession implements Fido2UserInterfaceSessi
   async confirmNewCredential({
     credentialName,
     userName,
+    userHandle,
     userVerification,
     rpId,
   }: NewCredentialParams): Promise<{ cipherId: string; userVerified: boolean }> {
```

Comment:
Since you return `undefined` below

```suggestion
  }: NewCredentialParams): Promise<{ cipherId?: string; userVerified: boolean }> {
```

Reply:
same for the `waitForUiChosenCipher` return

Reply:
typescript also doesn't like that `createCipher` can can `cipherService.encrypt` with a potentially undefined `activeUserId`